add-on/src/main/java/org/vaadin/crudui/form/AbstractAutoGeneratedCrudFormFactory.java

```diff
@@ -318,10 +318,11 @@ public void showError(CrudOperation operation, Exception e) {
         } else {
             if (CrudOperationException.class.isAssignableFrom(e.getClass())) {
                 // FIXME no Notification.Type
-                Notification.show(e.getMessage());
+                Notification.show(e.toString());
```

Comment:
`CrudOperationException`'s constructor requires a `String` that's set as the message. So a null check would be better here.